include/Basic/TokenKind.def

```diff
@@ -109,6 +114,30 @@ PUNCTUATOR(exclamation, '!')
 PUNCTUATOR(question, '?')
 PUNCTUATOR(stringQuote, '\"')
 PUNCTUATOR(singleQuote, '\'')
+PUNCTUATOR(at, '@')
+
+// Operators
+OPERATOR(plus, '+')
+OPERATOR(sub, '-')
+OPERATOR(mul, '*')
+OPERATOR(div, '/')
+OPERATOR(mod, '%')
+OPERATOR(eq, '==')
+OPERATOR(ne, '!=')
+OPERATOR(lt, '<')
+OPERATOR(le, '<=')
+OPERATOR(gt, '>')
+OPERATOR(ge, '>=')
+OPERATOR(and, '&&')
+OPERATOR(or, '||')
+OPERATOR(bitAnd, '&')
+OPERATOR(bitOr, '|')
+OPERATOR(bitXor, '^')
+OPERATOR(bitLShift, '<<')
+OPERATOR(bitRShift, '>>')
+OPERATOR(range, '...')
+OPERATOR(exclusiveRange, '..<')
```

Comment:
you didn't put the unary operators, where's `~`? exclamation should be here and not in punctuators as well